### DISCUSSION TITLE
💡 RFC: Go To Definition (F12) should  takes the respetive css file while path alias is active

### DISCUSSION THREAD

**Discussion by xkeshav** [withastro/language-tools#1029] [Feature requests]
### Background & Motivation

I have a .astro file and there import a css file  and I am using path alias ; configured in tsconfig.json, now in VS code when we hover over the import line and click on Go To Definiton File ( f12 ) ; it opens **node_modukles/astro/client.d.ts** file where declare syntax of cdd is menationed

```ts
// CSS
declare module '*.css' {
	const css: string;
	export default css;
}
```

while I want that it should open the respective file.

Note: this works fine when give relative path but alias path dint\\d not work, only for css file impot.

vs code 1.95
ubuntu 24.04
Astro V 4.16
Astro vs code extension v 2.15.4

### Help make it happen!

- [ ] I am willing to submit a PR to implement this change.
- [ ] I am willing to submit a PR to implement this change, but would need some guidance.
- [ ] I am not willing to submit a PR to implement this change.

**Comment by recursivezero** [comment]
Hello any update on this? when  can we do triage?

**Comment by xkeshav** [comment]
any help here?

**Comment by Princesseuh** [comment]
PRs are welcome. I currently have no time to dedicate to this. The current behavior is just TypeScript's default behavior, I think

**Comment by DarkRoku12** [comment]
Facing this issue, actually I googled `vscode f12 to css go to client.d.ts` which is the same problem but for Vite.
